### PR TITLE
[12.0][ADD] apriori: mpr_repair renamed models

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Also reordered them alphabetically.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza I think that now I did not leave any model rename 😅

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -26,15 +26,19 @@
...
     'product.uom.category': 'uom.category',
-    'hr.holidays': 'hr.leave',
-    'hr.holidays.status': 'hr.leave.type',
-    'sale.quote.template': 'sale.order.template',
-    'sale.quote.option': 'sale.order.template.option',
     'sale.quote.line': 'sale.order.template.line',
+    'sale.quote.option': 'sale.order.template.option',
+    'sale.quote.template': 'sale.order.template',
     'stock.incoterms': 'account.incoterms',
+    'stock.location.path': 'stock.rule',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/lib/apriori.py]: I don't know if this will give problems. Having two models renamed into a new one.

Comment (pedrobaeza) [reply odoo/addons/openupgrade_records/lib/apriori.py]: Yes. You should rename one of them, and merge the other one in this. Any way, is this model renaming being called in base module right now?

Comment (MiquelRForgeFlow) [reply odoo/addons/openupgrade_records/lib/apriori.py]: No, it's just only used in the openupgrade analysis.

Comment (pedrobaeza) [reply odoo/addons/openupgrade_records/lib/apriori.py]: The error would raise there, but any way, I think it's not a good idea to put both for the analysis.

Comment (MiquelRForgeFlow) [reply odoo/addons/openupgrade_records/lib/apriori.py]: Ok, I commented it instead.

**Pull request merged**